### DISCUSSION TITLE
Group update PRs based on file path

### DISCUSSION THREAD

**Discussion by tomkerkhove** [renovatebot/renovate#23892] [Request Help]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

Always latest

### If you're self-hosting Renovate, select which platform you are using.

Azure DevOps (dev.azure.com)

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

I'd love to have a PR for folderA and another one to update folderB

### What you tried so far.

I was hoping to use `includePaths` to achieve this but it looks like it's not working:
```json
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "major": false,
  "separateMajorMinor": true,
  "labels": ["dependencies"],
  "branchPrefix": "renovate/",
  "ignoreDeps": [],
  "printConfig": true,
  "includePaths": [
    "FolderA/**",
    "FolderB/**"
  ],
  "packageRules": [
    {
      "groupName": "Resiliency (FolderA)",
      "matchUpdateTypes": ["minor", "patch"],
      "matchPackagePrefixes": ["Polly"],
      "additionalBranchPrefix": "FolderA/",
      "includePaths": ["FolderA/**"]
    },
    {
      "groupName": "Resiliency (FolderB)",
      "matchUpdateTypes": ["minor", "patch"],
      "matchPackagePrefixes": ["Polly"],
      "additionalBranchPrefix": "FolderB/",
      "includePaths": ["FolderB/**"]
    }
  ],
  "extends": [
    "config:base",
    "docker:pinDigests"
  ]
}
```

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment]
`includePaths` is not correct for `packageFiles`. Instead, use `matchFileNames`
